src/environment_setup/python_package.ml

```diff
@@ -72,5 +72,5 @@ let default ~host ~run_program ~install_path () =
     create_python_tool ~host ~run_program ~install_path
       ~version:"0.0.21" (Pip, Package_PyPI "topiary");
     create_python_tool ~host ~run_program ~install_path
-      ~version:"0.2.0" (Pip, Package_PyPI "vaxrank");
+      ~version:"0.2.5" (Pip, Package_PyPI "vaxrank");
```

Comment:
This list defines what biokepi will pickup by default (with no cofig) when a user tries to use vaxrank.

So when a new vaxrank is realeased the question is "do we want it as the ‘default’".

I think for minor version bumps, yes, but for bigger changes we need to test and understand what they mean for the pipelines.